webscrambles/src/main/kotlin/org/worldcubeassociation/tnoodle/server/webscrambles/pdf/util/FontUtil.kt

```diff
@@ -11,7 +11,7 @@ object FontUtil {
     val NOTO_SANS_FONT: BaseFont = BaseFont.createFont("fonts/NotoSans-Regular.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED)
 
     const val MAX_SCRAMBLE_FONT_SIZE = 20f
-    const val MINIMUM_ONE_LINE_FONT_SIZE = 12f
+    const val MINIMUM_ONE_LINE_FONT_SIZE = 15f
```

Comment:
Please add a code comment with some (brief) form of reasoning why 15 was chosen. Seems pretty arbitrary otherwise.

Reply:
This is a Util class with a const val. The name should be self explanatory. 12f looks tiny to squeeze scrambles in one line.